BUILD

```diff
@@ -9,9 +9,15 @@ suitable build configuration is provided and will be used automatically if no ma
 is supplied - skip directly to running "make" (more details below) if you are on one of these
 systems and are happy to use the default configuration.
 
-For other systems, or to fine tune or correct the configuration, create and edit the
-"mconfig" file (start by copying one for a particular OS from the "configs" directory) to choose
-appropriate values for the configuration variables defined within. In particular:
+For other systems, or to fine tune or correct the configuration, you can use experimental
+"configure" script. This script try to find C++ compiler and auto-detects OS and checks some options
+are supported or not. Also this script have a interface to working with them, for more info use
+
+    ./configure --help
+
+command to see all options.
+otherwise you can create and edit the "mconfig" file (start by copying one for a particular OS
+from the "configs" directory) to choose appropriate values for the configuration variables defined within. In particular:
```

Comment:
This line has is too long, the file generally has lines < 100 characters in length